docs/source/features/memory_optimizations.rst

```diff
@@ -38,7 +38,7 @@ Selective Activation Recomputation
 """"""""""""""""""""""""""""""""""
 This method reduces memory footprint of activations significantly via smart activation checkpointing. This approach involves selectively storing only crucial activations and recomputing the others as needed. It is particularly useful in large models to minimize memory usage while controlling the computational cost.
 
-Refer to "Reducing Activation Recomputation in Large Transformer Models" for more details: https://arxiv.org/abs/2205.05198
+Refer to "Reducing Activation Recomputation in Large Transformer Models" for more details: https://arxiv.org/abs/2205.05198 .
```

Comment:
Delete the space after the 8 and before the period. Refer to "Reducing Activation Recomputation in Large Transformer Models" for more details: https://arxiv.org/abs/2205.05198.

Reply:
I am not expert here: seems removing this space will break the url link `https://arxiv.org/abs/2205.05198`

Reply:
Adding a period at the end of the link should not break it. I tested it and it worked for me.